### DISCUSSION TITLE
Claim being expired and being deleted.

### DISCUSSION THREAD

**Discussion by xmyoard** [GriefPrevention/GriefPrevention#1540] [General]
Claim being expired and being deleted.

How to prevent it in config?

**Comment by Jikoo** [comment]
Check your expiration settings, `GriefPrevention.Claims.Expiration`. You can increase the number of days offline before eligibility or decrease the total (bonus) claim block requirements for exemption.

**Reply by xmyoard** [reply]
Okay, that's right but in the config, there is a line:

Expiration:
ChestClaimDays: -1
UnusedClaimDays: -1

It was 30 days on previous I changed it to -1 will it cause any problem?
Anyway, thank you so much for helping me.

**Reply by Jikoo** [reply]
GP does not support -1 as disabled in most places. You're safer setting the minimum claim blocks to a small number so no one is eligible and setting the days to a very high number.

**Reply by xmyoard** [reply]
So what should I keep instead of -1 ? I mean I want the chest and Land claims to be forever

**Reply by Jikoo** [reply]
Choose a number players will not reasonably hit. 1000 days is nearly 3 years offline, for example.

**Reply by xmyoard** [reply]
Oh yeah, I thought It would crash the plugin, thanks for the support!
